app/server.py

```diff
@@ -90,18 +106,22 @@ def start_bot_process(self, bot_uuid):
         self._get_questions_queue(bot_uuid)
 
     def start_garbage_collector(self):
-        Timer(5*60.0, self.garbage_collector).start()
+        Timer(60.0, self.garbage_collector).start()
 
     def garbage_collector(self):
+        self._set_server_alive()
         with Lock():
             new_pool = {}
             for uuid, bot_instance in self._pool.items():
-                if not (datetime.now() - bot_instance['last_time_update']) >= timedelta(minutes=5):
+                if not (datetime.now() - bot_instance['last_time_update']) >= timedelta(minutes=60):
```

Comment:
The `time` on settings file here as well.

Reply:
I did had solved in next pull request, any problem if it come in next PR?